wasi/square/src/bindings.rs

```diff
@@ -2,12 +2,12 @@ pub type TaskQueueInput = lay3r::avs::types::TaskQueueInput;
 pub type Output = lay3r::avs::types::Output;
 #[doc(hidden)]
 #[allow(non_snake_case)]
-pub unsafe fn _export_run_task_cabi<T: Guest>(arg0: i64, arg1: *mut u8, arg2: usize) -> *mut u8 {
+pub unsafe fn _export_run_task_cabi<T: Guest>(arg0: u64, arg1: *mut u8, arg2: usize) -> *mut u8 {
     #[cfg(target_arch = "wasm32")]
     _rt::run_ctors_once();
     let len0 = arg2;
     let result1 = T::run_task(lay3r::avs::types::TaskQueueInput {
-        timestamp: arg0 as u64,
+        timestamp: Timestamp::from_nanos(arg0),
```

Comment:
I don't know enough about how these bindings are generated/used... @calvinrp  or @macovedj can one of y'all confirm the changes in this file are right?

Reply:
This file is autogenerated and shouldn't be modified manually. It is actually overwritten when you build the component.